python_lib.py

```diff
@@ -0,0 +1,657 @@
+import sys, os, ConfigParser, math, json, time, subprocess, commands, random, string, logging, logging.handlers, socket, StringIO
```

Comment:
Arash used this file to pickle and unpickle objects and the namespace needs to match to unpickle the current Pickle files. I removed unnecessary code except the objects that are being used. Once we create new pickle objects, this file can be deleted.